SUI/SUI/ContentView.swift

```diff
@@ -7,18 +7,166 @@
 
 import SwiftUI
 
+/// Экран плеера
 struct ContentView: View {
+    
+    // MARK: - Private enum
+    
+    private enum Constants {
+        static let trackImageWidth = 350
+        static let trackImageHeight = 400
+        static let trackImageCornerRadius = 17
+        static let floatDefaultValue = 0.0
+        static let buttonImageWidth = 100
+        static let buttonImageHeight = 50
+        static let buttonImageCornerRadius = 10
+        static let secondsInHour = 3600
+        static let secondsInMinute = 60
+        static let playImageName = "play.fill"
+        static let stopImageName = "stop.fill"
+        static let backwardImageName = "backward.fill"
+        static let forwardImageName = "forward.fill"
+        static let downloadButtonLabel = "Скачать"
+        static let downloadAlertText = "Скачать трек?"
+        static let shareButtonLabel = "Поделиться"
+        static let yesButtonText = "Да"
+        static let noButtonText = "Нет"
+        static let emptyString = ""
+        static let sliderMinimumValueText = "0:0"
+    }
+    
+    // MARK: - Body
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            setupSongImage
+            setupDurationSlider
+            setupSongNameLabel
+            setupPlayAndStopButtons
+            setupNextAndPreviosTrackButtons
+            setupDownloadAndShareButtons
+        }
+        .padding()
+    }
+    
+    // MARK: - ObserverObject private properties
+    @ObservedObject private var playerView = PlayerViewModel()
+    
+    // MARK: - State private properties
+    
+    @State private var progress: Float = 0
+    @State private var isSaveTrackButtonTapped = false
+    @State private var isShareTrackButtonTapped = false
+    
+    // MARK: - Private properties
+    
+    private var setupSongImage: some View {
```

Comment:
```suggestion
    private var songImageView: some View {
```
если нет парам это вычисляемое свойство называется как обычная вью без слова сетап в конце слово View